src/CADL.Extension/Emitter.Csharp/package.json

```diff
@@ -50,17 +50,17 @@
     },
     "dependencies": {
         "@autorest/csharp": "3.0.0-beta.20230309.1",
-        "@azure-tools/typespec-autorest": "0.28.0",
-        "@azure-tools/typespec-azure-core": "0.28.0",
-        "@azure-tools/typespec-client-generator-core": "0.28.0",
-        "@typespec/compiler": "0.42.0",
+        "@azure-tools/typespec-autorest": "0.29.0",
+        "@azure-tools/typespec-azure-core": "0.29.0",
+        "@azure-tools/typespec-client-generator-core": "0.29.0",
+        "@typespec/compiler": "0.43.0",
         "@typespec/eslint-config-typespec": "0.6.0",
-        "@typespec/eslint-plugin": "0.42.0",
-        "@typespec/http": "0.42.0",
-        "@typespec/library-linter": "0.42.0",
-        "@typespec/openapi": "0.42.0",
-        "@typespec/rest": "0.42.0",
-        "@typespec/versioning": "0.42.0",
+        "@typespec/eslint-plugin": "0.43.0",
+        "@typespec/http": "0.43.0",
```

Comment:
Can we update this to 0.43.1 which contains the bug fix for collection format.

Reply:
updated